src/fetch.ts

```
@@ -0,0 +1,264 @@
+    channels.fetchResponse.publish({
+      fetch: fetchMeta,
+      timingInfo: (state as any).timingInfo,
+      response: res!,
+    } as FetchResponseDiagnosticsMessage);
+    channels.response.publish({
+      request: reqMeta,
+      response: urllibResponse,
+    } as ResponseDiagnosticsMessage);
+    return res!;
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)